docs/api.md

```diff
@@ -192,8 +201,17 @@ Note that for CRDs which are "owned" by Antrea, `AntreaAgentInfo` and
 `AntreaControllerInfo`, resources are automatically created by the Antrea
 components using both API versions.
 
-All legacy API groups are planned for removal in December 2021. All versions of
-Antrea released after that will no longer include support for legacy API groups
-and will no longer ship with the mirroring controller. We recommend that all
-applications using the Antrea API be upgraded before then using the procedure
-detailed above.
+### Deleting legacy Kubernetes resources after an upgrade
+
+After a successful upgrade from Antrea < v1.6 to Antrea >= v1.6, you may want to
+manually clean up legacy Kubernetes resources which were created by an old
+Antrea version but are no longer needed. Note that keeping these resource will
+not impact any Antrea functions.
+
+To delete these legacy resources (CRDs and webhooks), run:
+
+```bash
+kubectl get crds -o=name --no-headers=true | grep "antrea\.tanzu\.vmware\.com" | xargs  kubectl delete
+kubectl get mutatingwebhookconfigurations -o=name --no-headers=true | grep "antrea\.tanzu\.vmware\.com" | xargs  kubectl delete
+kubectl get validatingwebhookconfigurations -o=name --no-headers=true | grep "antrea\.tanzu\.vmware\.com" | xargs  kubectl delete
```

Comment:
s/xargs  kubectl delete/xargs kubectl delete/g, one space after `xargs`

Reply:
I think we can also add a argument `-r` to `xargs`, like `xargs -r kubectl delete`. If there are not results, then the command will not run.

Reply:
Done